### DISCUSSION TITLE
加分看這裡

### DISCUSSION THREAD

**Discussion by lyykey** [1101-GS4719-C/Python-Community-GS4719-C#1914] [General]
不知道大家記不記得開學老師講的參加 AI 比賽能加分，以下為 2024 AI CUP 春季賽的比賽資訊，也歡迎大家分享其他比賽資訊。
歡迎大家有緣人，一起報名比賽~
以下為比賽網址：[比賽網址](https://www.aicup.tw/%E5%89%AF%E6%9C%AC-ai-cup-2023)
![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/105580604/52f44aee-0d4d-40b7-ad5d-2274e6ab32be)
![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/105580604/d5b9f559-dc47-4cbf-9da6-97bfc8b7b79d)
![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/105580604/edff6756-77d3-46f6-8fb6-570984909968)
![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/105580604/83a9aaca-f26b-4b0d-b691-2bdb33717729)
![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/105580604/5b52b2a6-8f6e-4510-8e01-3f72e3e161ae)

**Comment by benbebobo149** [comment]
這個比賽超棒的，我已經找好隊友組隊增加，不過今年的題目是有點困難。尤其是「跨相機多目標車輛追蹤組」，感覺要燒很多錢去訓練影像辨識。至於「生成式AI建構無人機於自然環境之地形導航競賽」，完全不懂它想要我們做甚麼事情。希望之後可以找機會切磋討論，或許可以從不同的角度來理解這些題目，共同探索解決的可能性。最後，我要感謝同學分享這個讓我有機會參與刺激比賽的機會。

**Comment by KH1126** [comment]
最近也在尋找關於Ai相關的競賽。
目前還有在網上找到另一個比賽
[https://bhuntr.com/tw/competitions/2024-ee-creative-challengeaiot-7lzw0za116aokeutrt]()
但看起來也相當有難度。
因為之前沒有做過關於機械學習或深度學習相關程式專案，所以最近也在學其中的相關知識。
希望可以在最近學習好相關知識，最後可以實作出一些關於深度學習相關的專案出來。
